integration-tests/src/tests/client/process_blocks.rs

```diff
@@ -4511,3 +4511,162 @@ mod contract_precompilation_tests {
         assert!(caches[1].get(&contract_key.0).unwrap().is_none());
     }
 }
+
```

Comment:
Can you please add some bugs (one by one in most interesting places) in the chunk_cache_nodes and make sure that those integration tests break?

Reply:
You can also try to fully disable chunk caching to see if those tests break.